src/main/java/net/sf/jabref/exporter/ExportFormats.java

```diff
@@ -223,7 +219,7 @@ public void actionPerformed(ActionEvent e) {
                     Globals.prefs.put(JabRefPreferences.EXPORT_WORKING_DIRECTORY, file.getParent());
 
                     final File finFile = file;
-                    final Set<String> finEntryIDs = entryIds;
+                    final List<BibEntry> finEntries = entries;
```

Comment:
Further down it is used in an anonymous method, which is only possible if it final.